### DISCUSSION TITLE
Person Card

### DISCUSSION THREAD

**Discussion by zsarnett** [home-assistant/frontend#11282] [Other feature requests]
## The request

![image](https://user-images.githubusercontent.com/18730868/89740627-00919080-da50-11ea-8d97-d7250d54f4be.png)

The card would be able to take a person entity and auto-fill in the Image, battery, and location if available.

It would then work as an entities card on the bottom where you supply entities that you want on the card and it would display them.

This is a quick mock-up of what I think it should look like. Looking for feedback on if people would use this or no? As well as feedback on the design.

## The alternatives

![image](https://user-images.githubusercontent.com/18730868/89740659-464e5900-da50-11ea-924e-7e8f053906ff.png)

Creating a picture entities card and doing it basically the same way

**Comment by SeanPM5** [comment]
This looks great! I would definitely use this and I think many others would as well. I do something kind of similar with the custom button card but yours looks way nicer 😃 

One thing I think would be neat to add is [Proximity integration](https://www.home-assistant.io/integrations/proximity/) support in this. So if the person is home, it could just say "Home" like your mockup, but if they're away it could have secondary text that says "5 miles away" or "Driving towards home" or some additional context like that.

**Comment by bramkragten** [comment]
I also like the comment by @thomasloven to make it a header, so it can be used above an entities card

**Comment by zsarnett** [comment]
That may be the route, but I would want it as a stand alone card for ease of use. That was more talking about you supply it entities and it displays them. Reason for that is from my mock the state is actually the text after the icon not the name of the entity. As the entity should be related to the person I dont think having the entity name is very usefull

**Comment by balloob** [comment]
Here is what I had mocked up for the config redesign. Just another idea:

<img width="1057" alt="image" src="https://user-images.githubusercontent.com/1444314/96641192-a735bf80-1324-11eb-8b19-555acac8b40a.png">

I like the first screenshot a lot, but it would also require a cover picture.

**Comment by SondreNjaastad** [comment]
Hi! I started making this card. Its my first lovelace card, so i bet there is a handfull of tweaks to be made. I allso wanted to indicate MS Teams status on the card.

[https://github.com/SondreNjaastad/person-card](https://github.com/SondreNjaastad/person-card) 

The card is atm fully functional and you can add as many entites in the card as you want
![20201115_214450](https://user-images.githubusercontent.com/3602435/99196772-c4787500-278e-11eb-84fa-4a00d9957ddb.jpg)
![20201115_214422](https://user-images.githubusercontent.com/3602435/99196777-c80bfc00-278e-11eb-8ec6-7feae6940f1f.jpg)

**Comment by zsarnett** [comment]
Nice. Looks great!

**Comment by SondreNjaastad** [comment]
I think i want to remove the circle around the image and add a dot in the lower right corner instead

**Comment by spacegaier** [comment]
How does it look if there is no background image defined? Will you fill this header section with a theme color (e.g. the teal in your case)? Or will we provide a default HA image that is used as default so the users can see that it actually contains an image and therefore realize they can switch it out with a custom one?

**Comment by SondreNjaastad** [comment]
> How does it look if there is no background image defined? Will you fill this header section with a theme color (e.g. the teal in your case)? Or will we provide a default HA image that is used as default so the users can see that it actually contains an image and therefore realize they can switch it out with a custom one?

I have not yet handeled default background image. It currently just stays white. Using a theme color is a great idea. i allso need to implement theme on the rest of the card.

**Comment by SondreNjaastad** [comment]
I believe this looks better than the ring around the image

![Screenshot_20201116-090424_Home Assistant](https://user-images.githubusercontent.com/3602435/99226859-c40dc780-27ea-11eb-987d-29c52924c5b9.jpg)

**Comment by balloob** [comment]
We need some more space for the location. Like if it is something like "University" or "Grocery Shopping"

**Comment by balloob** [comment]
It's not quite clear to me what the red dot stands for ?

**Comment by spacegaier** [comment]
> 
> 
> It's not quite clear to me what the red dot stands for ?

I think it's meant to show an availability status, here derived from the MS Teams status.
Perhaps this is something that could be configurable to select the source for that information or the user can not use that option.

**Comment by SondreNjaastad** [comment]
I have now set up a release action on github.
The card in the image is in a single cloumn next to another of the same card, so this is at 50% width.
The status indicator is optional 😊

**Comment by jorblad** [comment]
Is it possible to use a templated image url so that I could have one banner when home and another when at work for example?

**Comment by balloob** [comment]
Uh, it's a custom card and not a contribution to core? We should stop talking about it right now then. This issue is about a person card **in core**.

**Comment by zsarnett** [comment]
I was thinking about this card recently. Thinking about Core Functionality does it make sense to have a Header image? or just the profile image. Thinking about redesigning this and making this completely automatically built

Probably something very similar to Paulus's design. But with a few extra pieces of information

**Comment by zsarnett** [comment]
To keep it simple I am thinking that Paulus's Design is best. Cover image may be too much for core.

![image](https://user-images.githubusercontent.com/18730868/136082089-2203db90-c830-45ec-8bb2-daac00f704e5.png)


@matthiasdebaat let me know what you think for this. In the Frontend Figma under Lovelace

**Comment by SondreNjaastad** [comment]
> To keep it simple I am thinking that Paulus's Design is best. Cover image may be too much for core.
> 
> ![image](https://user-images.githubusercontent.com/18730868/136082089-2203db90-c830-45ec-8bb2-daac00f704e5.png)
> 
> @matthiasdebaat let me know what you think for this. In the Frontend Figma under Lovelace

This looks great! Feel free to use my repo for Development of this. No need to keep what is the now

**Comment by matthiasdebaat** [comment]
> To keep it simple I am thinking that Paulus's Design is best. Cover image may be too much for core.
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/18730868/136082089-2203db90-c830-45ec-8bb2-daac00f704e5.png)
> 
> 
> 
> 
> 
> @matthiasdebaat let me know what you think for this. In the Frontend Figma under Lovelace

That looks nice! Clean and consistent with the HA interface. Could you share the Figma URL?

**Comment by balloob** [comment]
https://www.figma.com/file/4LZqQZRynDZGTgA2rN35Te/Home-Assistant-config-mockup?node-id=44%3A243
